src/test/java/dk/cngroup/lentils/service/StatusServiceTest.java

```diff
@@ -164,6 +164,22 @@ public void getStatusScoreForOneLockedCyper() {
         assertEquals(0, service.getStatusScore(status.getTeam(), status.getCypher()));
     }
 
+    @Test
```

Comment:
Tady bych těch testů napsal víc pro všechny ty případy:
1/ Když je šifra posledni
2/ Když není poslední, ale ostatní jsou locked
3/ Když náledující šifry nejsou locked.

Reply:
A když se na to tak ještě dívám, možná bych to napsal jako integrační test bez těch mocků.
Uložit všechno normálně do databáze a testovat co ta metoda vrátí.

Reply:
nejake jsem dopsala, tak snad OK

Reply:
A jo, toto jsem přehlédl, i když je to o řádek výš.
Žádný podstatný důvod to asi nemá. Ty boxed datové typy se používají spíš, když s tím potřebuješ pracovat jako s objektem, předávat reference atd. Jinak to moc smysl nemá.